test/dlc_tests/conftest.py

```diff
@@ -85,6 +85,9 @@
     "huggingface_mxnet_inference",
     "huggingface_tensorflow_trcomp_training",
     "huggingface_pytorch_trcomp_training",
+    # Stability
+    "stability_pytorch_inference",
```

Comment:
I think this should be `stabilityai_pytorch_inference` with the "AI". As of now, this fails to run on SAI images since the repo name does not match the fixture.

Reply:
Yeah it has to match the repo name, will make the change